build-tools/packages/bundle-size-tools/.eslintrc.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
     plugins: ["@typescript-eslint"],
     parser: "@typescript-eslint/parser",
     parserOptions: {
-        project: ["./tsconfig.json", "./src/test/tsconfig.json"],
+        project: ["./tsconfig.json"],
```

Comment:
Copy and paste error caught by Fluid-Build.  ('bundle-size-tools' does not have tests)